### DISCUSSION TITLE
Known issues with Parallel VDS

### DISCUSSION THREAD

**Discussion by hyoklee** [HDFGroup/hdf5#3672] [Announcements]
These issues are from HDFFV-10287:

- "printf" style mappings are not supported.
- The VDS must be opened collectively.
- When using separate source file(s), the source file(s) cannot be opened by the library while the VDS is open.
- When using separate source file(s), data cannot be written through the VDS unless the mapping is equivalent to 1 process per source file.
- All I/O is independent internally (possible performance penalty).
- Each rank does an independent open of each source file it accesses (possible performance penalty).

**Comment by derobins** [comment]
Is this just a documentation request or is this a feature implementation/improvement request?
